.pre-commit-config.yaml

```
@@ -25,3 +25,14 @@ repos:
     rev: 3.9.2
     hooks:
       - id: flake8
+  - repo: https://github.com/nbQA-dev/nbQA
+    rev: 1.5.0
```

> I'll fix this upstream, will push out a 1.5.1

done 🚀 

should be enough to just do:
```suggestion
    rev: 1.5.1
```

---

Thanks! I see that 1.7.0 is already out :)